include/swift/Basic/Sanitizers.h

```diff
@@ -21,6 +21,8 @@ enum class SanitizerKind : unsigned {
   Thread,
 };
 
+enum class SanitizerCoverageKind : unsigned { None = 0, Function, BB, Edge };
```

Comment:
@jrose-apple It originally was on multiple lines but clang-format decided it knew better. I could change it but if someone clang formats the file in the future it is going to become a single line declaration again.